src/main/java/seedu/address/logic/commands/LogCommand.java

```diff
@@ -0,0 +1,34 @@
+package seedu.address.logic.commands;
+
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+/**
+ * Saves the results of a FindCommand to the logbook.
+ */
+public class LogCommand extends Command {
+
+    public static final String COMMAND_WORD = "log";
+
+    public static final String COMMAND_WORD_ALIAS = "lo";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + "or" + COMMAND_WORD_ALIAS
+            + ": Saves the results of the most recent FindCommand to the logger tab.\n"
+            + "Example: " + COMMAND_WORD + "or" + COMMAND_WORD_ALIAS;
```

Comment:
Same as the comments in `ClearLogCommand` and `AppendLogCommand`.
```suggestion
            + "Example 1: " + COMMAND_WORD + "\n"
            + "Example 2: " + COMMAND_WORD_ALIAS
```